packages/item/theme/lumo/vaadin-item-styles.js

```diff
@@ -62,7 +63,7 @@ const item = css`
 
   /* Disabled */
   :host([disabled]) {
-    color: var(--lumo-disabled-text-color);
+    color: var(--_disabled-value-color);
     cursor: default;
```

Comment:
As with Details summary, this is not a field value, so let's not use the new prop here.